app/views/dashboards/index.html.erb

```diff
@@ -1,21 +1,29 @@
 <h1>Little Esty Shop</h1> 
 <p><%= @merchant.name %></p>
 
-<div class="items-divider">
+<div id="items-divider">
   <li><%= link_to "My Items", merchant_items_path %></li>
 </div>
 
-<div class="invoices-divider">
+<div id="invoices-divider">
   <li><%= link_to "My Invoices", merchant_invoices_path %></li>
 </div>
 
-<div class="fav-customers-divider">
+<div id="fav-customers-divider">
   <h2>Favorite Customers</h2>
 
     <% @customers.each do |customer| %>
-      <div class="customer-info">
+      <div id="customer-info">
         <p><%= customer.full_name %></p>
         <p><%= customer.transaction_count %></p>
       </div>
     <% end %>
+</div>
+
+<div id="ready-to-ship">
+  <h2>Items Ready To Ship</h2>
+    <p><% @items.each do |item| %></p>
+      <p><%= item.name %></p>
+      <p><%= link_to "#{item.invoice_id}", merchant_invoice_path(@merchant, item.invoice_id) %></p>
```

Comment:
what is `.invoice_id`? i pulled your branch locally and don't see an instance method for the item model. if you are looking for the invoice_id you would need to call `item.invoice_item.invoice_id` unless there is something i am missing. also small nit the quotes and interpolation aren't needed you can just do `link_to item.invoice_id`

Reply:
oh are you getting it from the `ready_to_ship` method?